[2022-01-29 14:08:17,513] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-12-02T06:00:00+00:00 [queued]>
[2022-01-29 14:08:17,548] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-12-02T06:00:00+00:00 [queued]>
[2022-01-29 14:08:17,548] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:08:17,548] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-29 14:08:17,548] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:08:17,600] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2021-12-02 06:00:00+00:00
[2022-01-29 14:08:17,606] {standard_task_runner.py:52} INFO - Started process 441 to run task
[2022-01-29 14:08:17,614] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2021-12-02T06:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmpl6pv08s9', '--error-file', '/tmp/tmpbrdzkkmv']
[2022-01-29 14:08:17,614] {standard_task_runner.py:77} INFO - Job 53: Subtask get_data
[2022-01-29 14:08:17,775] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-12-02T06:00:00+00:00 [running]> on host d1e1fe005577
[2022-01-29 14:08:17,985] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-02T06:00:00+00:00
[2022-01-29 14:08:17,987] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 14:08:17,989] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "helloworld!!"']
[2022-01-29 14:08:18,003] {subprocess.py:85} INFO - Output:
[2022-01-29 14:08:18,006] {subprocess.py:89} INFO - helloworld!!
[2022-01-29 14:08:18,006] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 14:08:18,073] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20211202T060000, start_date=20220129T140817, end_date=20220129T140818
[2022-01-29 14:08:18,124] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 14:08:18,239] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
